src/Symfony/Bridge/Doctrine/Tests/Validator/DoctrineLoaderTest.php

```diff
@@ -38,7 +37,9 @@ class DoctrineLoaderTest extends TestCase
 {
     protected function setUp(): void
     {
-        if (!trait_exists(AutoMappingTrait::class)) {
+        $validator = Validation::createValidatorBuilder();
+
+        if (!method_exists($validator, 'addDefaultDoctrineAnnotationReader')) {
```

Comment:
I do not understand why I had to do this, probably because of dependency changes, but without this it will fails with lowest tests.

Reply:
This test is not really related to your changes, is it? Can you revert it? What is the error that you get?

Reply:
This is why I told I don't understand why this fail, if I revert this test, it will fail as it was failing before, see https://travis-ci.com/github/symfony/symfony/jobs/439068912 for last CI fail.

I think the version bump in validator for property-info component might have effect here ?
(see https://github.com/symfony/symfony/pull/39020/files#diff-f7fc46a53267414c8ac26ce9319845a6a8affd51847ad4a9ab9953fff6e46eceL41)

Reply:
It looks like the subtree split of the validator component was outdated when the CI ran. I think you can simply revert this change and should be good.

Reply:
It was like that since a week at least but its working now (like, I ran multiple CI runs with the same output everytime and I did reproduce it locally), I really don't understand this :confused:

I'm gonna stash my commits ~

Reply:
Other failing builds (for examples):
- https://travis-ci.com/github/symfony/symfony/jobs/430664219
- https://travis-ci.com/github/symfony/symfony/jobs/430664261